docs/en/setup/backend/dynamic-config.md

```diff
@@ -8,6 +8,7 @@ Right now, SkyWalking supports following dynamic configurations.
 |:----:|:----:|:----:|
 |receiver-trace.default.slowDBAccessThreshold| Thresholds of slow Database statement, override `receiver-trace/default/slowDBAccessThreshold` of `applciation.yml`. | default:200,mongodb:50|
 |receiver-trace.default.uninstrumentedGateways| The uninstrumented gateways, override `gateways.yml`. | not set |
+|alarm.default.alarm-settings| The alarm settings, will override `alarm-settings.yml`. | not set |
```

Comment:
Above two lines document has issues. The column requires `Value Format Example`, not default value. We could give user a link or sample data to show how to config it.

I think I missed that for gateway setting too.